nuke-from-orbit/loadtester

```diff
@@ -221,22 +221,31 @@ set_oauth_secret() {
 }
 
 # Set the Looker username and password secret
+# dry run trick is to handle updates in place
 set_looker_secret() {
   if ! [[ -z $looker_user || -z $looker_pass ]]
   then
     kubectl create secret generic website-creds \
       --from-literal=username=$looker_user \
-      --from-literal=password=$looker_pass
+      --from-literal=password=$looker_pass \
+      --dry-run=client \
+      --save-config \
+      -o yaml \
+      | kubectl apply -f -
```

Comment:
Very handy, I still don't quite understand this pattern, I'm going to go read about it...